# Robot name goes here
mobile_manipulator:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # gazebo_ros_control:
  #   pid_gains:
  #     rear_left_shaft_joint:
  #       p: 10
  #       i: 0.01 
  #       d: 0
  #     rear_right_shaft_joint: 
  #       p: 10 
  #       i: 0.01
  #       d: 0

  rear_motor_left: 
    joint: rear_left_shaft_joint
    pid: {p: 70, i: 3, d: 10.0}
    type: velocity_controllers/JointVelocityController
  rear_motor_right: 
    joint: rear_right_shaft_joint
    pid: {p: 70, i: 3, d: 10.0}
    type: velocity_controllers/JointVelocityController
  left_castor_motor: 
    joint: front_left_castor_joint
    pid: {p: 70, i: 3, d: 10.0}
    type: effort_controllers/JointPositionController
  right_castor_motor: 
    joint: front_right_castor_joint
    pid: {p: 70, i: 3, d: 10.0}
    type: effort_controllers/JointPositionController

  first_link_motor: 
    joint: first_joint
    pid: {p: 70, i: 3, d: 10.0}
    type: position_controllers/JointPositionController
  second_link_motor: 
    joint: second_joint
    pid: {p: 70, i: 3, d: 10.0}
    type: position_controllers/JointPositionController
  third_link_motor: 
    joint: third_joint
    pid: {p: 70, i: 3, d: 10.0}
    type: position_controllers/JointPositionController

  fourth_link_motor: 
    joint: fourth_joint
    pid: {p: 70, i: 3, d: 10.0}
    type: position_controllers/JointPositionController
  fifth_link_motor: 
    joint: fifth_joint
    pid: {p: 70, i: 3, d: 10.0}
    type: position_controllers/JointPositionController
  sixth_link_motor: 
    joint: sixth_joint
    pid: {p: 70, i: 3, d: 10.0}
    type: position_controllers/JointPositionController

 

  

 